SasquatchMac/SasquatchMac/ViewControllers/TransmissionViewController.swift

```diff
@@ -473,7 +473,7 @@ class TransmissionViewController: NSViewController, NSTableViewDataSource, NSTab
     let propertyIndex = getCellRow(forTextField: sender)
     let target = TransmissionViewController.targetsShared!.transmissionTargets[selectedTarget!]!
     propertyValues[selectedTarget!]![propertyIndex] = sender.stringValue
-    let value = sender.stringValue.isEmpty ? nil as String? : sender.stringValue
+    let value = sender.stringValue.isEmpty ? nil as String! : sender.stringValue
```

Comment:
It doesn't look correct

Reply:
May be something like:
`let value: String? = sender.stringValue.isEmpty ? nil : sender.stringValue`